client/src/components/shared/GeneralPagination/index.js

```diff
@@ -37,29 +38,34 @@ const getNextPage = ({currentPage, page, from, to, pageCount}) => {
 }
 
 const GeneralPagination = ({
-  pageCount, maxPages, currentPage, onPaginationClick,
+  pageCount,
+  maxPages,
+  currentPage,
+  onPaginationClick,
 }: PaginationProps) => {
   const pageShowCount = Math.min(maxPages, pageCount)
   const {from, to} = getPageRange({pageShowCount, currentPage, pageCount})
   return (
     <Pagination className="GeneralPagination">
-      {
-        range(from, to).map((page) => (
-          <PaginationItem
-            key={page}
-            onClick={() => onPaginationClick && onPaginationClick(
-              getNextPage({currentPage, page, from, to, pageCount})
-            )}
+      {range(from, to).map((page) => (
+        <PaginationItem
+          key={page}
+          onClick={() =>
```

Comment:
again, was here in previous commit, but you shouldn't pass arrow functions to onClick of react components

that said, this is the pagination that will get a rewrite, so no problem for keeping it for now, just take note